Blazor/README.md

```diff
@@ -125,12 +123,19 @@ To use, in your project:
     ...    
 ```
   - If using the application template, simply use `appTop` and `appFooter` instead
+  - When calling InstallCDTS in Program.cs, one of the optional parameters is a boolean that needs to be set to true if using the application template
+
+## RefTop/RefFooter
+
+ - By default the user does not need to configure the refTop and refFooter objects for the layout. If required, this can be done in Program.cs through the SetupBase object.
+
+ - When calling the InstallCDTS in Program.cs, it can take an optional SetupBase object which you can use to define the RefTop and RefFooter properties such as ExitScript.
 
```

Comment:
RefTop/RefFooter are CDTS concepts that doesn't really need to be mentioned here.  The person reading this will likely not be familiar with CDTS anyway.  BUT the second point (about InstallCDTS) should be moved to the Program.cs section (and maybe an example added)

Reply:
done